components/FilAriane.tsx

```diff
@@ -0,0 +1,51 @@
+import Link from 'next/link'
+import { useEffect, useState } from 'react'
+
+import ChevronIcon from '../assets/icons/chevron_right.svg'
+
+interface FilArianeProps {
+  currentPath: string
+}
+
+export default function FilAriane({ currentPath }: FilArianeProps) {
+  const [ariane, setAriane] = useState<{ fragment: string; href: string }[]>([])
+
+  useEffect(() => {
+    const crumbs: { fragment: string; href: string }[] = []
+    currentPath
+      .split('/')
+      .slice(1, -1)
+      .forEach((fragment, index) => {
+        if (index > 0) {
+          crumbs.push({
+            fragment,
+            href: `${crumbs[index - 1].href}/${fragment}`,
+          })
+        } else {
+          crumbs.push({ fragment, href: `/${fragment}` })
+        }
+      })
+    setAriane(crumbs)
+  }, [currentPath])
+
+  return (
+    <ul className='mb-2 flex items-center'>
```

Comment:
```suggestion
<nav role="navigation" aria-label="breadcrumbs"
    <ol className='mb-2 flex items-center'>
```
on encapsule d'une balise nav, en tant que navigation secondaire. L'aria-label avec "breadcrumbs" par convention, et liste ordonnée

Reply:
La balise `<nav>` n'a pas déjà un rôle "navigation" implicite ?

Reply:
On peut pas dire "fil d'ariane" au lieu de "breadcrumbs" ?